packages/react-core/src/components/Spinner/Spinner.tsx

```diff
@@ -13,13 +13,15 @@ export interface SpinnerProps extends Omit<React.HTMLProps<HTMLSpanElement>, 'si
   /** Additional classes added to the Spinner. */
   className?: string;
   /** Size variant of progress. */
-  size?: 'sm' | 'md' | 'lg' | 'xl';
+  size?: 'sm' | 'md' | 'lg' | 'xl' | 'inline';
   /** Text describing that current loading status or progress */
   'aria-valuetext'?: string;
   /** Whether to use an SVG (new) rather than a span (old) */
   isSVG?: boolean;
   /** Diameter of spinner set as CSS variable */
   diameter?: string;
+  /** @beta Indicates the spinner is inline and the size should inherit the text font size. */
```

Comment:
I would probably mention that the size prop is ignored if this is set.

Reply:
Is it OK that `isInline` overrides `size` here, but in the icon component, `size` will override `isInline`?

Reply:
Seems like it should be uniform - unless there is a good reason not to be. Although potentially "breaking" if we change the icon component?

Reply:
I missed this comment.  Yes.  I would expect consistent API's across both components.  @mcoker is it possible to update the `Icon` implementation.  It is still a beta feature.

Reply:
@tlabaj yep, or we can update spinner in v5. Another thing we can do is have "inline" be the default. The default is currently `xl` - I think it's less likely that you'd want an `xl` spinner over having the spinner match whatever the current text size is.

FWIW, currently `pf-m-sm` and `pf-m-inline` have the same role, which is why one overrides the other. t-shirt sizing just doesn't work for a `1em/[whatever the text size is]` size. If there is confusion around what overrides what, is there a reason we would want to separate them? In other words, could they be values of the same enum that controls the size?

This is _inline_ with the core issue about the standard size variations and "marketing" (aka "display", aka `.pf-m-display-[size]`), the role of those classes, and if they should be values of a standard "size" prop if they have the same role and you can't combine them.

Reply:
@mcoker.  If we want to combine them into one prop, I would suggest calling it `1em` rather than inline since that is the "size".  I think combining them would be best and more clear.

Reply:
The problem with `1em` is that 1) it's more literal than conceptual since we may want icons or whatever used inline with content be +/-2px or something, and 2) "inline" may do other things like adjust the thing vertically or whatever so that it's as plug 'n play as it can be to put it beside other things. For example, the icon's "inline" variant also changes the color to match whatever the text color is where it's used. 

Though 2 makes me think "inline" isn't necessarily just a size, but we have other uses of "size" variants that make adjustments to the content in some of the sizes but not others.

Reply:
@mcoker can we open a follow up issue to update the `Icon`?

Reply:
@tlabaj I added this to the size naming issue open for v5 - https://github.com/patternfly/patternfly/issues/5155